src/Models/Post.php

```diff
@@ -214,7 +214,22 @@ public static function getForm()
                                 })
                                 ->minDate(now()->addMinutes(5))
                                 ->native(false),
+                            DateTimePicker::make('published_at')
```

Comment:
In general, `published_at` value refers to the time at which post has been published. We believe such value should be set automatically by the application and manually setting such value doesn't align with the scope of the application. Can you elaborate the use case of this implementation ?

Reply:
In most cases, one might want to schedule the publication using a scheduling system. However, there are also situations where immediate publication is required, bypassing the queues. In my specific case, I have authors who schedule their posts for future dates and others who need to publish immediately.

Therefore, it would be beneficial to have the option to manually set the published_at value to accommodate both scenarios: scheduled posts and immediate publications. This flexibility would ensure that the system can adapt to the diverse needs and workflows of our users.